backend/src/main/java/com/aljoschazoeller/backend/person/domain/NewPersonDTO.java

```diff
@@ -0,0 +1,11 @@
+package com.aljoschazoeller.backend.person.domain;
+
+import jakarta.validation.constraints.NotBlank;
+
+public record NewPersonDTO(
+        @NotBlank(message = "First name must have at least one non-whitespace character")
+        String firstName,
+        @NotBlank(message = "Last name must have at least one non-whitespace character")
+        String lastName
```

Comment:
Hier könnte man auch noch die max länge validieren (https://github.com/dead-claudia/github-limits)

Reply:
Danke für den Hinweis., ist eine gute Idee. Person hat nichts it GitHub zutun, daher wähle ich 50 Buchstaben wie bei uns intern in den Anforderungen.

Reply:
Alles klar :)